ui/tests/acceptance/tools-test.js

```diff
@@ -7,9 +7,6 @@ moduleForAcceptance('Acceptance | tools', {
   beforeEach() {
     return authLogin();
   },
-  afterEach() {
-    return authLogout();
-  },
```

Comment:
Why this change? I don't follow.

Reply:
Oops, I thought I was seeing a race condition and took this out, but it was something else and it'd be better to have this in especially now that we're switching users.